app/DashBoard/Presenters/templates/Blocks/favicons.latte

```diff
@@ -0,0 +1,8 @@
+{block favicon}
+<link rel="apple-touch-icon" sizes="180x180" href="/favicons/apple-touch-icon.png">
```

Comment:
A je to praktické, třeba teď budu duplikovat kód, vytvářet komponentu 2x, či ne?